packages/shared-base/src/NextID/index.ts

```diff
@@ -0,0 +1,22 @@
+import { NextIDPlatform, ProfileIdentifier } from '../index.js'
+import { EnhanceableSite } from '../Site/type.js'
+
+export function nextIDPlatformToNetwork(platform: NextIDPlatform) {
+    // inline the pairs for testing, if out the pairs will not work
+    const pairs = [[NextIDPlatform.Twitter, EnhanceableSite.Twitter]]
+    const pair = pairs.find((x) => x[0] === platform)
+    return pair?.[1]
+}
+
+export function networkToNextIDPlatform(site: EnhanceableSite) {
```

Comment:
Please use `createLookupTableResolver` stead.

Reply:
I refactor in this pr: #7229